src/web/app/pages-session/session-submission-page/session-submission-page.component.ts

```diff
@@ -788,8 +800,12 @@ export class SessionSubmissionPageComponent implements OnInit, AfterViewInit {
         );
       }
 
-      if (!isQuestionFullyAnswered) {
+      if (!isQuestionMaxAdhered) {
+        notYetMaxAnswersAdhered.add(questionSubmissionFormModel.questionNumber);
+      } else if (!isQuestionFullyAnswered) {
         notYetAnsweredQuestions.add(questionSubmissionFormModel.questionNumber);
+      } else if (isQuestionPartiallyAnswered) {
+        notFullyAnsweredQuestions.add(questionSubmissionFormModel.questionNumber);
```

Comment:
I believe that there is a bug with the logic here.

Scenario: I only have 1 question. `maximum number of students each respondent should give feedback to` is set to 3. I have selected an option for each of the student i'm giving feedback to but in the modal i see `Note that some questions allow for more answers. They are: 1.`. Looking at the logic, this shows up when this condition is met `notFullyAnsweredQuestions.length !== 0`.

I believe it comes from L748 where once the condition is met `isQuestionPartiallyAnswered` will always stay true.

![image](https://user-images.githubusercontent.com/46486515/229296194-6772fb5c-8a3d-4965-9b3e-763c84ecfbd0.png)